larray/session.py

```diff
@@ -368,14 +369,15 @@ def load(self, fname, names=None, engine='auto', display=False, **kwargs):
         # TODO: support path + *.csv
         if engine == 'auto':
             _, ext = os.path.splitext(fname)
-            engine = ext_default_engine[ext.strip('.')]
+            engine = ext.strip('.') if '.' in ext else 'csv'
```

Comment:
name of variable ('engine') is misleading. Should be ext or stripped_ext or normalized_ext or whatever.